docs/ref/contrib/postgres/aggregates.txt

```diff
@@ -125,7 +125,8 @@ General-purpose aggregation functions
 .. class:: JSONBAgg(expressions, distinct=False, filter=None, default=None, ordering=(), **extra)
 
     Returns the input values as a ``JSON`` array, or ``default`` if there are
-    no values.
+    no values. When a ``JSON`` array is returned, you can query using 
+    :lookup:`key or index based transformations <jsonfield.key>` on the result.
```

Comment:
```When a ``JSON`` array is returned``` is unnecessary this aggregation **always** returns a `JSON` array or `None`. Please see my suggestion https://github.com/django/django/pull/14640#discussion_r675350993,

Reply:
my intention to not mix this with the case when `None` or another default value is returned. Apparently, the chaining works even in that case. simplified the sentence. thanks, for the feedback.